kedro_neptune/__init__.py

```diff
@@ -410,9 +412,13 @@ def __init__(self):
     def after_catalog_created(
             self,
             catalog: DataCatalog,
-            run_id: str,
+            save_version: str,
     ) -> None:
-        self._run_id = hashlib.md5(run_id.encode()).hexdigest()
+        print("🤡", save_version)
```

Comment:
?

Reply:
I wanted to remember about fixme below. Do you think datetime `save_version` works and I can remove both print and comment?